src/math/trigonometry.js

```diff
@@ -156,24 +176,54 @@ p5.prototype.atan2 = function(y, x) {
 };
 
 /**
- * Calculates the cosine of an angle. This function takes into account the
- * current <a href="#/p5/angleMode">angleMode</a>. Values are returned in the range -1 to 1.
+ * Calculates the cosine of an angle. `cos()` is useful for many geometric
+ * tasks in creative coding. The values returned oscillate between -1 and 1.
```

Comment:
Maybe "the values returned oscillate between -1 and 1 as the input value increases" would be clearer?

Reply:
Good suggestion! Added this for `sin()` as well.